### PR TITLE
Testpage

### PR DESCRIPTION
**Pull request by LukasPedersen**
+Added Testpage.cshtml

### CODE REVIEW

File: ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace ITGuru.FourWheels.Web.Pages
```
Comment (ghost) [inline ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml.cs]: Should we use the .NET 6 method of writting namespaces instead?

File: ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+﻿@page
+@model ITGuru.FourWheels.Web.Pages.TestpageModel
+@{
```
Comment (jeongoks) [inline ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml:3]: Why is there no HTML/CSS added to the site? How are you able to test this?

**Review by jeongoks — COMMENTED** [review]
Check comment on the file.

File: ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml (hunk 2 above)
Comment (LukasPedersen) [reply ITGuru.FourWheels/ITGuru.FourWheels.Web/Pages/Testpage.cshtml:3]: We dont need HTML/CSS!

**Review by jeongoks — APPROVED** [review]
Looks finno.

**Pull request merged**